### PR TITLE
fix input box issue for safari and ipnone

### PR DESCRIPTION
**Pull request by PunyaChapagain**
if you check internet explorer search box in safari or iPhone you cannot type or select input field. this fix the issue

### CODE REVIEW

**Comment by ShizukuIchi** [comment]
Hey @PunyaChapagain!
Thank you! I didn't notice this! But I think  `*:not(input):not(textarea)` is better.

**Comment by PunyaChapagain** [comment]
Thank you for your suggestion and I have implemented inside container and works perfectly

**Comment by ShizukuIchi** [comment]
Hi @PunyaChapagain. That works great. Please fix other lines to meet origin code style then I'll merge it.

File: src/WinXP/index.js [hunk 1]
```diff
@@ -79,8 +79,8 @@ const reducer = (state, action = { type: '' }) => {
           state.apps.length > 1
             ? FOCUSING.WINDOW
             : state.icons.find(icon => icon.isFocus)
-            ? FOCUSING.ICON
-            : FOCUSING.DESKTOP,
+              ? FOCUSING.ICON
```
Comment (ShizukuIchi) [inline src/WinXP/index.js]:
```suggestion
            ? FOCUSING.ICON
```

File: src/WinXP/index.js [hunk 2]
```diff
@@ -79,8 +79,8 @@ const reducer = (state, action = { type: '' }) => {
...
+              : FOCUSING.DESKTOP,
```
Comment (ShizukuIchi) [inline src/WinXP/index.js]:
```suggestion
            : FOCUSING.DESKTOP,
```

File: src/WinXP/index.js [hunk 3]
```diff
@@ -362,8 +362,8 @@ const Container = styled.div`
   background: url(https://i.imgur.com/Zk6TR5k.jpg) no-repeat center center fixed;
   background-size: cover;
   animation: ${({ state }) => animation[state]} 5s forwards;
-  * {
-    user-select: none;
+  *:not(input):not(textarea) {
+    user-select: none;
```
Comment (ShizukuIchi) [inline src/WinXP/index.js]:
```suggestion
    user-select: none;
```

**Pull request merged**